src/main/webapp/find-events.html

```diff
@@ -63,52 +63,52 @@ <h1 id="find-events-title">find events</h1>
 
           <span class="modal-span">
-            <p id="modal-time-text">date:</p>
-            <input type="number" id="modal-month" name="month" placeholder="mm" min="1" max="12" step="1">
+            <p id="modal-time-text" tabindex="0">date:</p>
+            <input type="number" id="modal-month" name="month" placeholder="mm" min="1" max="12" step="1" tabindex="0" aria-label="month 2 digit">
             <p id="modal-slash">/</p>
-            <input type="number" id="modal-day" name="day" placeholder="dd" min="1" max="31" step="1">
+            <input type="number" id="modal-day" name="day" placeholder="dd" min="1" max="31" step="1" tabindex="0" aria-label="day 2 digit">
           </span>
 
           <span class="modal-span">
-            <p id="modal-time-text">start:</p>
-            <input type="number" id="modal-start-hour" name="startHour" placeholder="h" min="1" max="12" step="1">
-            <input type="number" id="modal-start-minute" name="startMinute" placeholder="m" min="0" max="60" step="1">
-            <select id="start-am-or-pm" name="startAMorPM">
-              <option value="am" selected>am</option>
-              <option value="pm">pm</option>
+            <p id="modal-time-text" tabindex="0">start:</p>
+            <input type="number" id="modal-start-hour" name="startHour" placeholder="h" min="1" max="12" step="1" tabindex="0" aria-label="start hour">
+            <input type="number" id="modal-start-minute" name="startMinute" placeholder="m" min="0" max="60" step="1" tabindex="0" aria-label="start minute">
+            <select id="start-am-or-pm" name="startAMorPM" aria-label="start am or pm?" role="listbox">
+              <option value="am" aria-hidden="false" role="option" aria-label="am">am</option>
+              <option value="pm" aria-hidden="false" role="option" aria-label="pm">pm</option>
             </select>
           </span>
           
           <span class="modal-span">
-            <p id="modal-time-text">end:</p>
-            <input type="number" id="modal-end-hour" name="endHour" placeholder="h" min="1" max="12" step="1">
-            <input type="number" id="modal-end-minute" name="endMinute" placeholder="m" min="0" max="60" step="1">
-            <select id="end-am-or-pm" name="endAMorPM">
-              <option value="am" selected>am</option>
-              <option value="pm">pm</option>
+            <p id="modal-time-text" tabindex="0">end:</p>
+            <input type="number" id="modal-end-hour" name="endHour" placeholder="h" min="1" max="12" step="1" tabindex="0" aria-label="end hour">
+            <input type="number" id="modal-end-minute" name="endMinute" placeholder="m" min="0" max="60" step="1" tabindex="0" aria-label="end minute">
+            <select id="end-am-or-pm" name="endAMorPM" aria-label="end am or pm?" aria-haspopup="true" role="listbox">
```

Comment:
This am/pm `select` has `aria-haspopup="true"`, but the other one doesn't; i would either fix or mention why this discrepancy exists

Reply:
Good catch! Thanks